packages/manager/modules/pci/src/projects/project/private-registry/create/create.html

```diff
@@ -25,7 +25,9 @@
                     data-name="agreements"
                     data-required>
                     <span data-translate="pci_project_new_description_project_contracts"
-                        data-translate-values="{ contractsLinks: $ctrl.getCompiledLinks('<a href=\'<%= pdf %>\' target=\'_blank\' rel=\'noopener\'><%= name %></a>') }">
+                        data-translate-values="{
+                            contractsLinks: $ctrl.getCompiledLinks('<a class=\'oui-link_icon\' href=\'<%= pdf %>\' target=\'_blank\' rel=\'noopener\'><%= name %><span class=\'oui-icon oui-icon-external_link\' aria-hidden=\'true\'></span></a>')
+                        }">
```

Comment:
Given the translation https://github.com/ovh-ux/manager/blob/50b74d939bb088bb1c89c64a6f65abfa7fe5de74/packages/manager/modules/pci/src/projects/new/description/translations/Messages_fr_FR.json#L3
Maybe we could just extract links as a `<ul>` instead of having to pass them through `translate-values`. Does that make sense ? 

Moreover your module shoud bring its own translation, so you should have a `translations` folder under `packages/manager/modules/pci/src/projecrs/project/private-registry/create` containing the translations needed for the `create` module

Reply:
Can move the translation to the create module but changing to `<ul>` wont make it appear in same line.